packages/playground/interactive-block-playground/vite.config.ts

```diff
@@ -0,0 +1,106 @@
+/// <reference types="vitest" />
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
+import viteTsConfigPaths from 'vite-tsconfig-paths';
+// eslint-disable-next-line @nrwl/nx/enforce-module-boundaries
+import ignoreWasmImports from '../ignore-wasm-imports';
+// eslint-disable-next-line @nrwl/nx/enforce-module-boundaries
+import {
+	websiteDevServerHost,
+	websiteDevServerPort,
+	remoteDevServerHost,
+	remoteDevServerPort,
+} from '../build-config';
+// eslint-disable-next-line @nrwl/nx/enforce-module-boundaries
+import virtualModule from '../vite-virtual-module';
+
+const proxy = {
+	'^/plugin-proxy.*&artifact=.*': {
+		target: 'https://playground.wordpress.net',
+		changeOrigin: true,
+		secure: true,
+	},
+	'/plugin-proxy': {
```

Comment:
For later – a standard vite config with these endpoints in place would be super useful